Src/Plugins/Security/SEC101_053.GoogleServiceAccountKeyValidator.cs

```diff
@@ -0,0 +1,117 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+
+using Google.Apis.Auth.OAuth2;
+
+using Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.Utilities;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security
```

Comment:
>Security [](start = 62, length = 8)

Hello! Let's talk offline. I don't think you can fit this class of validation into our model.